salt/states/boto_vpc.py

```diff
@@ -131,19 +126,35 @@ def present(name, cidr_block, instance_tenancy=None, dns_support=None,
 
 
 def absent(name, tags=None, region=None, key=None, keyid=None, profile=None):
+    '''
+    Ensure VPC with passed properties is absent.
+
+    name
```

Comment:
Actually, never mind. I thought this was a new function, but you're just adding the docs for it here. Apologies!